app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -8,7 +10,21 @@ enum class EventCategory {
   SPORTS,
   CONFERENCE,
   EXHIBITION,
-  COMMUNITY
+  COMMUNITY,
+  PARTY,
+  UNDEFINED;
```

Comment:
These my clash with #125

Reply:
Right ty, I will discuss it with Christine